OWNERS_ALIASES

```diff
@@ -29,11 +29,11 @@ aliases:
     - MushuEE
     - spiffxp
   ci-signal-reporter-reviewers:
-    - leonardpahlke # 1.24 CI Signal Lead
-    - voigt # 1.24 CI Signal Shadow
-    - lauralorenz # 1.24 CI Signal Shadow
-    - shuheiktgw # 1.24 CI Signal Shadow
-    - nivedita-coder # 1.24 CI Signal Shadow
+    - Nivedita-coder # 1.26 CI Signal Lead
+    - shuheiktgw # 1.26 CI Signal Shadow
+    - mehabhalodiya # 1.26 CI Signal Shadow
+    - Namanl2001 # 1.26 CI Signal Shadow
+    - bharitwal # 1.26 CI Signal Shadow
```

Comment:
Since `ci-reporter` is tooling and needs people who have in-depth knowledge of the code for reviews, there should be a set of people who are not rotated across cycles.

I don't think we should rotate everyone in this reviewer group every cycle. For example, the membership to this reviewer group can be defined as Past reviewers of this area + CI Signal Leads + Shadows who show substantial contributions consistently to `ci-reporter`.

@kubernetes/sig-release-leads -- what do you think?

Reply:
Can we create a new group in k8s.io to be linked here?

Reply:
Would the new group be the CI Signal team?

Reply:
I see two benefits to rotating the CI signal team as reviewers 1. it should lead to the shadows getting them more involved with release tooling 2. reduce the load on the release leads/managers. 

I think we can also keep contributors as reviewers / approvers after a cycle

Reply:
Commit history of contributors: https://github.com/kubernetes/release/commits/master/cmd/ci-reporter

Reply:
Yes, we should keep contributors who consistently contributed and not rotate them.